R/rd-html.R

```diff
@@ -114,15 +114,21 @@ as_html.tag_link <- function(x, ..., index = NULL, current = NULL) {
     # \link[=dest]{name}
     link_local(in_braces, substr(opt, 2, nchar(opt)), index = index, current = current)
   } else {
-    match <- regexec('([^:]+):(.*)', opt)
-    parts <- regmatches(opt, match)[[1]]
+    match <- regexec('^([^:]+)(?:|:(.*))$', opt)
+    parts <- regmatches(opt, match)[[1]][-1]
 
-    if (length(parts) == 0) {
+    pkg <- attr(current, "pkg")
+    stopifnot(!is.null(pkg))
```

Comment:
@krlmlr The test fails (#243) because 'current' is NULL here, and does not have an attribute 'pkg'. No idea what 'current' is supposed to do here.